.gitignore

```diff
@@ -17,3 +17,4 @@ types/extension-renderer-api.d.ts
 extensions/*/dist
 docs/extensions/api
 site/
+.prettierrc
```

Comment:
That's the reason that it's listed in `.gitignore`.  Prettier does lot more than `eslint` and I'm trying to find a way to use it to format code without messing up the existing code so that those who want to utilize it, can do so.

Reply:
I agree with @jansav, current `eslint` configuration is a tad suboptimal, and causes manual work to keep code reasonably formatted. If optional usage of `prettier` even complies to `eslint`, then why not go for it?